### PR TITLE
Winter

### PR DESCRIPTION
**Pull request by Abubakarharuna10**
## Checklist before submitting a proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by Abubakarharuna10** [comment]
.

**Comment by exu3** [comment]
@Abubakarharuna10 could you please update the pull request description to include a description for your project along with your Hack Club Slack username?

**Comment by exu3** [comment]
Oh sorry I might clarify—this is the PR description.

Screenshot 2022-12-25 at 7 51 18 PM

File: Abubakarharuna10.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Plan
+1) Having requirements to use in this project.
+2) Hardware components,
+  -NVIDIA Jetson Nano Developer Kit
+  -Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter
+  -CanaKit 5V 2.5A Power Supply
+  -Raspberry Pi Camera Module V2
+  -Anker PowerCore 5000
+   -PX4 Pixhawk
+3) Implimentation
```
Comment (exu3) [inline Abubakarharuna10.md:31]: In your plan, could you describe how the components fit together and what steps you would take to do that? Ideally, someone reading this document would be able to imagine what the final product would look like.

File: Abubakarharuna10.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+name: "Abubakarharuna10"
+
+project: "Artificial Rescuer"
+
+Slack Username "@abubakarharuna"
+
+---
```
Comment (exu3) [inline Abubakarharuna10.md:8]: Small thing: could you edit this header section to match the format of the example? 

It should follow this format:

```
---
name: "@Abubakarharuna10"
project: "Artificial Rescuer"
---
```

**Review by exu3 — CHANGES_REQUESTED** [review]
Thanks for submitting a project proposal, @Abubakarharuna10. 

I think the idea is great; the part where it's lacking right now is the `Plan` section. I left some comments below; let me know if you have any questions!

**Comment by Abubakarharuna10** [comment]
I update the file

**Comment by Abubakarharuna10** [comment]
> @Abubakarharuna10 could you please update the pull request description to include a description for your project along with your Hack Club Slack username?

Yes I have done that now

File: Abubakarharuna10.md (hunk 2 above)
Comment (Abubakarharuna10) [reply Abubakarharuna10.md:8]: Sure!

File: Abubakarharuna10.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  -Anker PowerCore 5000
+   -PX4 Pixhawk
+3) Implimentation
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi Camera Module V2.1  |https://thepihut.com/products/raspberry-pi-camera-module| 34.35 |
+| NVIDIA Jetson Nano Developer Kit |https://www.sparkfun.com/products/16271| $149.00|
```
[the remark this answers is not in the document]
Comment (Abubakarharuna10) [reply Abubakarharuna10.md:37]: Yes, It's!

File: Abubakarharuna10.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  -Raspberry Pi Camera Module V2
+  -Anker PowerCore 5000
+   -PX4 Pixhawk
+3) Implimentation
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi Camera Module V2.1  |https://thepihut.com/products/raspberry-pi-camera-module| 34.35 |
```
[the remark this answers is not in the document]
Comment (Abubakarharuna10) [reply Abubakarharuna10.md:39]: Alright

File: Abubakarharuna10.md (hunk 1 above)
Comment (Abubakarharuna10) [reply Abubakarharuna10.md]: I update it again!

File: Abubakarharuna10.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Abubakarharuna10) [reply Abubakarharuna10.md:39]: check out this link    https://www.amazon.com/Official-Raspberry-Pi-8-megapixel-XYGStudy/dp/B01G01JJJ0/ref=sr_1_2?crid=J818MRHDX9ED&keywords=Raspberry+Pi+Camera+Module+V2.1&qid=1672020961&sprefix=raspberry+pi+camera+module+v2.1%2Caps%2C355&sr=8-2

[the remark this answers is not in the document]
Comment (Abubakarharuna10) [reply Abubakarharuna10.md]: Another alternative for Nvidia dev tool kit too https://www.amazon.com/NVIDIA-Jetson-Nano-Developer-945-13450-0000-100/dp/B084DSDDLT/ref=sr_1_3?crid=1M6KCN83IMQGA&keywords=NVIDIA+Jetson+Nano+Developer+Kit+%28V3%29&qid=1672021508&sprefix=nvidia+jetson+nano+developer+kit+v3+%2Caps%2C762&sr=8-3

File: Abubakarharuna10.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+---
+name: "@Abubakarharuna10"
+project: "Artificial Rescuer"
+---
+
+# Project Name
+Artificial Rescue
```
Comment (exu3) [inline Abubakarharuna10.md:7]:
```suggestion
# Artificial Rescuer
```

File: Abubakarharuna10.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+to do the actual object-detection (people) in the camera's view While the UAV is flying a waypoint mission using ArduPilot, PX4, or any other autonomous flight control stack, the absolute location of people in the camera view can be calculated based on the altitude, orientation, and GPS location of the UAV.
+
+## Plan
+
+I will use  NVIDIA Jetson Nano Developer Kit to run computer vision algorithms to detect people as the vehicle flies overhead along with an Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter that will allow me to connect to the Jetson Nano and monitor processes while the vehicle is in a flight setting then the CanaKit 5V 2.5A Power Supply will provide a stable power source while setting up the Jetson Nano and the Raspberry Pi Camera Module V2 will be mounted to the underside of the vehicle to capture video of the ground below then Anker PowerCore 5000 This power pack was selected for its 2A output and compact size. then the PX4 Pixhawk on an autonomous drone or airplane will communicate with the Jetson Nano over a wired MAVLink connection.
```
Comment (exu3) [inline Abubakarharuna10.md:23]:
```suggestion
I will use  NVIDIA Jetson Nano Developer Kit to run computer vision algorithms to detect people as the vehicle flies overhead. I will use an Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter that will allow me to connect to the Jetson Nano and monitor processes while the vehicle is in a flight setting. The CanaKit 5V 2.5A Power Supply will provide a stable power source while setting up the Jetson Nane. The Raspberry Pi Camera Module V2 will be mounted to the underside of the vehicle to capture video of the ground below. As forthe Anker PowerCore 5000, this power pack was selected for its 2A output and compact size. Then, the PX4 Pixhawk on an autonomous drone or airplane will communicate with the Jetson Nano over a wired MAVLink connection.
```

Suggestion: I added punctuation to make it easier to read.

File: Abubakarharuna10.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+1) Having requirements to use in this project.
+2) Hardware components,
+  -NVIDIA Jetson Nano Developer Kit
+  -Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter
+  -CanaKit 5V 2.5A Power Supply
+  -Raspberry Pi Camera Module V2
+  -Anker PowerCore 5000
+   -PX4 Pixhawk
+3) Implimentation
```
Comment (exu3) [inline Abubakarharuna10.md:33]: Seems like this part can be removed as the above section mentions all of this in more detail.
```suggestion
```

**Review by exu3 — CHANGES_REQUESTED** [review]
Thanks for adding more detail to the `Plan` section. 

It's not quite clear to me what you plan to make the vehicle out of. Did you have any plans for that?

File: Abubakarharuna10.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (exu3) [reply Abubakarharuna10.md]: Ok! Make sure it ships to the country you live in.

File: Abubakarharuna10.md (hunk 6 above)
Comment (Abubakarharuna10) [reply Abubakarharuna10.md]: Sure Thanks for your suggestion!

File: Abubakarharuna10.md (hunk 7 above)
Comment (Abubakarharuna10) [reply Abubakarharuna10.md]: Yes, I have done that now!

Thank You!

**Comment by cfanoulis** [comment]
Hi @Abubakarharuna10! Thanks for your proposal. It's been a while since we've had any news from you and we want to know if you're still interested in the grant. Please reply in the next 48 hours to this PR (or this email) if you're still interested, else we'll be closing this proposal. Hacky New Year!

**Comment by Abubakarharuna10** [comment]
> 

okay

File: Abubakarharuna10.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi Camera Module V2.1  |https://www.amazon.com/Official-Raspberry-Pi-8-megapixel-XYGStudy/dp/B01G01JJJ0/ref=sr_1_2?crid=J818MRHDX9ED&keywords=Raspberry+Pi+Camera+Module+V2.1&qid=1672020961&sprefix=raspberry+pi+camera+module+v2.1%2Caps%2C355&sr=8-2| 34.35 |
+| NVIDIA Jetson Nano Developer Kit |https://www.amazon.com/NVIDIA-Jetson-Nano-Developer-945-13450-0000-100/dp/B084DSDDLT/ref=sr_1_3?crid=1M6KCN83IMQGA&keywords=NVIDIA+Jetson+Nano+Developer+Kit+%28V3%29&qid=1672021508&sprefix=nvidia+jetson+nano+developer+kit+v3+%2Caps%2C762&sr=8-3| $149.00|
+| Anker PowerCore 5000 | https://www.amazon.com/Anker-PowerCore-Ultra-Compact-High-Speed-Technology/dp/B072QD7F6N?th=1  | $34.95 |
+| Edimax   |https://www.amazon.com/Edimax-EW-7611ULB-Wi-Fi-Bluetooth-Adapter/dp/B01KVZB3A4 | $12.11  |
+| Total           |                                  | $230.41 |
```
Comment (exu3) [inline Abubakarharuna10.md:38]: Do these products ship to Nigeria?

File: Abubakarharuna10.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+## Plan
+
+
+For the first day I will make all the neccessary equipment I would need to use. and the next 2 or 3 days I will write the programming code and build the AI model .
+The rest of the days will be spent constructing and testing the project.
+
+I will use  NVIDIA Jetson Nano Developer Kit to run computer vision algorithms to detect people as the vehicle flies overhead. I will use an Edimax EW-7811Un 150Mbps 11n Wi-Fi USB Adapter that will allow me to connect to the Jetson Nano and monitor processes while the vehicle is in a flight setting. The CanaKit 5V 2.5A Power Supply will provide a stable power source while setting up the Jetson Nane. The Raspberry Pi Camera Module V2 will be mounted to the underside of the vehicle to capture video of the ground below. As for the Anker PowerCore 5000, this power pack was selected for its 2A output and compact size. Then, the PX4 Pixhawk on an autonomous drone or airplane will communicate with the Jetson Nano over a wired MAVLink connection  but you can use any autonomous drone or airplane in order to Mount your work on it.
```
Comment (exu3) [inline Abubakarharuna10.md:26]: Do you have an autonomous drone or airplane to mount your project on? No worries if you don't. I am curious as to what your final project might look like. Will you be mounting your system onto a drone, or just building the detection system?

**Review by exu3 — CHANGES_REQUESTED** [review]

File: Abubakarharuna10.md (hunk 8 above)
Comment (Abubakarharuna10) [reply Abubakarharuna10.md:38]: yes

File: Abubakarharuna10.md (hunk 9 above)
Comment (Abubakarharuna10) [reply Abubakarharuna10.md:26]: yes I will try to own it before ending of the  project else I will collect my friend's own to use for demonstration

Thank you!

**Review by exu3 — APPROVED** [review]

**Review by daisybanks — APPROVED** [review]
@Abubakarharuna10 this sounds so awesome and so helpful! I'm dming you now on Slack to talk about next steps in getting you set up on Bank and getting you your grant! :)

**Comment by cfanoulis** [comment]
hey @Abubakarharuna10, congrats on the grant :) We've been trying to reach you for your ~car's extended warranty~ bank onboarding call - the next step to bring you close to your grant. However, your banker, Daisy, has been having difficulty reaching you to schedule a call. Can you please check [Slack](https://hackclub.slack.com) & reply to their DM?

a bank onboarding call is crucial for the grant process, as this is where you'll be seeing Hack Club Bank for the first time (unless you have a demo account!). Also, you'll go through the Fiscal Sponsorship contract - a legal document officially getting you on da hack bank

Please make sure to schedule a meeting within the next 2 days (that is, until Jan 26th) - else, your grant will be forfeited! (and nobody wants that). If you're having trouble, please get in touch with either me or your banker via Slack.

Toodaloo!

**Comment by Abubakarharuna10** [comment]
> hey @Abubakarharuna10, congrats on the grant :) We've been trying to reach you for your ~car's extended warranty~ bank onboarding call - the next step to bring you close to your grant. However, your banker, Daisy, has been having difficulty reaching you to schedule a call. Can you please check [Slack](https://hackclub.slack.com) & reply to their DM?
> 
> a bank onboarding call is crucial for the grant process, as this is where you'll be seeing Hack Club Bank for the first time (unless you have a demo account!). Also, you'll go through the Fiscal Sponsorship contract - a legal document officially getting you on da hack bank
> 
> Please make sure to schedule a meeting within the next 2 days (that is, until Jan 26th) - else, your grant will be forfeited! (and nobody wants that). If you're having trouble, please get in touch with either me or your banker via Slack.
> 
> Toodaloo!

Alright, Thank you so much!

**Pull request merged**